src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -76,6 +76,18 @@
                DisplayName="Annotations" />
   </EnumProperty>
 
+  <BoolProperty Name="Prefer32Bit"
+                DisplayName="Prefer 32-bit"
+                Description="Set the 32-bit preferred flag for the application. Only applies to executables."
+                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2166468"
+                Category="General">
+    <BoolProperty.Metadata>
+      <NameValuePair Name="VisibilityCondition">
+        <NameValuePair.Value>(has-net-framework)</NameValuePair.Value>
```

Comment:
Doesn't Prefer32Bit only make sense with `AnyCPU`? I don't think it has any effect if the platform target has any other value (for example, `x86`, `x64`, `ARM`, ...). The visibility condition should probably be extended to include that also.

Reply:
That is something I couldn't get a concrete explanation on. There is clearly a `anycpu32bitpreferred` platform, but I don't know if this flag is directly related to that platform. I forgot to link this originally in the description of this PR:
https://dzone.com/articles/what-anycpu-really-means-net

Basically, `32bitpref` is just a flag on the metadata of the executable. This properly might link directly to setting that flag. Why I suspect this is because one would think that we would actually set the value `anycpu32bitpreferred` in the PlatformTarget property. In this PR, I linked to that documentation, but thinking about it now, linking to the flags documentation might be more appropriate.

Reply:
Added to visibility condition.